smetrics/src/test/scala/com/evolutiongaming/smetrics/MeasureDurationSpec.scala

```diff
@@ -24,6 +30,48 @@ class MeasureDurationSpec extends AnyFunSuite with Matchers {
     duration shouldEqual 2.nanos
     state shouldEqual State.Empty
   }
+
+  test("MeasureDurationOps.measured") {
+    val handler: FiniteDuration => StateT[Unit] =
+      duration => StateT { state =>
+        val timestamp = duration.toNanos
+        timestamp shouldEqual 5
+        (state, timestamp)
+      }.void
+    val (state, _) = Timer[StateT]
+      .sleep(5.nanos)
+      .measured(handler)
+      .run(State.Empty)
+    state shouldEqual State.Empty
+  }
+
+  test("MeasureDurationOps.measuredCase success") {
+    val successHandler = mock[FiniteDuration => IO[Unit]]
```

Comment:
Please remove mockito, this could be smth like:
```
val test = for {
  mvar <- MVar.empty[IO, FiniteDuration]
  _        <- IO.pure(()).measured(mvar.put)
  arg    <- mvar.tryTake
  } yield assert(arg.nonEmpty)
  ```
  Also no need to sleep in tests, imo, measure duration is already tested and it works